custom/icds_reports/reports/poshan_progress_dashboard_data.py

```diff
@@ -32,25 +48,19 @@ def calculate_percentage_single_row(row, truncate_out=True):
     return row
 
 
-def calculate_aggregated_row(data, aggregation_level, data_period, unique_id):
+def calculate_aggregated_row(data, aggregation_level):
```

Comment:
Removing the prepare quarter dict function as we are doing it at the initial level only. Rest of the function is just same(also not taking aggregated output in case of the month, it will help in making things more generic as well as we are using same thing in the PPR so maintain consistency also)